shared/actions/chat/inbox.js

```diff
@@ -228,13 +228,15 @@ function _toSupersedeInfo(
 function* _processConversation(c: RPCChatTypes.InboxUIItem): Generator<any, void, any> {
   const conversationIDKey = c.convID
 
-  // Update reset participants
-  yield Saga.put(
-    ChatGen.createUpdateResetParticipants({
-      conversationIDKey,
-      participants: c.resetParticipants || [],
-    })
-  )
+  // Update reset participants (only implicit teams)
+  if (c.membersType === RPCChatTypes.commonConversationMembersType.impteam) {
+    yield Saga.put(
+      ChatGen.createUpdateResetParticipants({
+        conversationIDKey,
+        participants: c.resetParticipants || [],
```

Comment:
Maybe clean to do `c.membersType === RPCChatTypes.commonConversationMembersType.impteam ? c.resetParticipants : []` to drop the indent from the `if`?

Reply:
or I guess the whole thing would get rendered with no users then?

Reply:
this just updates the reset field, which if its not an implied team just isn't represented at all